README.zh-CN.md

```
@@ -1,352 +1,214 @@
+    }
+  },
+  handler: async (args) => {
+    // 这里编写你的业务逻辑
+    return { content: [{ type: 'text', text: `用户 ${args.userId} 的信息...` }] }
+  }
+})
 ```
 
-**工作流程：**
+✅ **完成！** 你的应用现在就是一个 MCP 服务。
+你可以将其连接到任何 MCP 兼容的客户端，或使用我们的 [TinyRemoter](#-添加-ai-遥控器可选) 直接与应用对话。
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)